src/resolvers/workspace.js

```diff
@@ -320,12 +323,57 @@ module.exports = {
       }
 
       const planModel = await factories.plansFactory.findById(planId);
+      const oldPlanModel = await factories.plansFactory.findById(workspaceModel.plan);
+      const userModel = await factories.usersFactory.findById(user.id);
 
       if (!planModel) {
         throw new UserInputError('Plan with passed ID doesn\'t exists');
       }
 
-      await workspaceModel.changePlan(planModel._id);
+      try {
+        // Charge money for new plan
+        const transaction = await accounting.purchase({
+          accountId: workspaceModel.accountId,
+          amount: planModel.monthlyCharge,
+          description: 'Monthly charge',
```

Comment:
это надо вынести в константу. 

```suggestion
          description: 'Monthly charge for the workspace plan',
```

Reply:
не думаете, что тут имеет смысл в описании указать, что это списание при смене плана?

@GeekaN2 @khaydarov @nikmel2803

Reply:
А это даже не совсем ежемесячно списание, скорее просто списание за месяц при смене плана. Тогда не думаю, что есть смысл в константу выносить, ведь такое описание только здесь будет встречаться